src/types.ts

```diff
@@ -61,7 +71,7 @@ export interface PaginationColors {
 }
 export interface TableThemeColors {
   tableBackgroundColorFromTheme: string;
-  backgroundColor: string | undefined;
+  backgroundColor?: string;
```

Comment:
Are you sure this is the same thing? This is set to undefined sometimes, not unset. It might be the same thing but I am not sure

Reply:
undefined happens when we don’t assign a value to a variable, so unset === set to undefined

Reply:
ok